libraries/entities/src/EntityTree.cpp

```diff
@@ -2213,6 +2213,7 @@ void EntityTree::fixupNeedsParentFixups() {
 
             entity->postParentFixup();
         } else if (getIsServer() || _avatarIDs.contains(entity->getParentID())) {
+            std::lock_guard<std::mutex> lock(_childrenOfAvatarsLock);
```

Comment:
I *think* the problem is that _childrenOfAvatars was modified by different threads, so this just adds a lock in the few places it's used to prevent that